addons/static/templates/policy_cache_transformer_persona.json

```diff
@@ -1368,47 +1368,63 @@
       "actions": ["add-relationship", "remove-relationship"]
     }
   ],
-  "persona-abac-delete": [
+  "persona-api-create-abac": [
```

Comment:
What does this govern

Reply:
It's corresponding to `persona-api-create` which allows creating assets with match criteria. In atlas policies it will also only to create child assets because QN needs to match with selected assets but in ABAC it will allow creating any assets that matches the criteria provided. Had raised this with product team, they are okay with this.